qiskit_experiments/library/characterization/qubit_spectroscopy.py

```diff
@@ -74,7 +74,7 @@ def _default_experiment_options(cls) -> Options:
     def _default_analysis_options(cls) -> Options:
         """Default analysis options."""
         options = super()._default_analysis_options()
-        options.result_parameters = [ParameterRepr("freq", "f01", "Hz")]
+        options.result_parameters = [ParameterRepr("freq", "freq", "Hz")]
```

Comment:
Why this change is necessary? rabi EF returns `f12`, so notation will become different. You may also want to update rabi EF.

Reply:
Good point see here: 8b3bb92